src/pages/Home/SwitchTxType.tsx

```diff
@@ -3,8 +3,8 @@ import type { FC } from "react";
 import { useTranslation } from "react-i18next";
 
 export enum TxType {
-  ONCHAIN,
-  LIGHTNING,
+  ONCHAIN = 0,
```

Comment:
Same as above enum

Reply:
https://biomejs.dev/linter/rules/use-enum-initializers/

Can disable it if wanted